### PR TITLE
makes ice regent melt when heated

### PR DESCRIPTION
**Pull request by TechDragon610**
🆑 TechDragon610
rscadd: ice regent melts
/:cl:

makes sure that the ice regent turns to water when heated.

### CODE REVIEW

File: code/modules/reagents/oldchem/chemical_reaction/chemical_reaction_food.dm [hunk 1]
```diff
@@ -109,7 +109,15 @@
...
 	mix_message = "Ice forms as the water freezes."
 	mix_sound = null
-
+/datum/chemical_reaction/water
+	name = "Water"
+	id = "water"
+	result = "water"
+	required_reagents = list("ice" = 1)
+	result_amount = 1
+	max_temp = 274
```
Comment (LittleBigKid2000) [inline code/modules/reagents/oldchem/chemical_reaction/chemical_reaction_food.dm]: This should be min_temp, not max_temp.

File: code/modules/reagents/oldchem/chemical_reaction/chemical_reaction_food.dm [hunk 2]
```diff
@@ -109,7 +109,15 @@
...
+	mix_message "Water pools as the ice melts."
```
Comment (TheDZD) [inline code/modules/reagents/oldchem/chemical_reaction/chemical_reaction_food.dm]: You forgot the = sign here, which is why Travis is failing.

**Comment by TechDragon610** [comment]
fixed but why on earth is there conflicts when it worked fine on my end?

**Comment by TechDragon610** [comment]
fixed. (again, must have been an unseen typo.)

**Comment by FalseIncarnate** [comment]
You know, this would have been a sensible thing to add when I made it so you could freeze water into ice. No clue why I didn't add it then. :+1:

**Comment by Fox-McCloud** [comment]
:+1:

File: code/modules/reagents/oldchem/chemical_reaction/chemical_reaction_food.dm [hunk 3]
```diff
@@ -109,7 +109,15 @@
...
-
+/datum/chemical_reaction/water
+	name = "Water"
+	id = "water"
+	result = "water"
+	required_reagents = list("ice" = 1)
+	result_amount = 1
+	min_temp = 274
+	mix_message = "Water pools as the ice melts."
+	mix_sound = null
```
Comment (Fox-McCloud) [inline code/modules/reagents/oldchem/chemical_reaction/chemical_reaction_food.dm]: space here, please.

Comment (TechDragon610) [inline code/modules/reagents/oldchem/chemical_reaction/chemical_reaction_food.dm]: where?

Comment (TechDragon610) [inline code/modules/reagents/oldchem/chemical_reaction/chemical_reaction_food.dm]: like so?

Comment (Fox-McCloud) [inline code/modules/reagents/oldchem/chemical_reaction/chemical_reaction_food.dm]: yes. Thank you.

**Comment by Vivalas** [comment]
:-1:  for not modelling vapor pressure and temperature for all regents

**Comment by TheDZD** [comment]
Pull Request is missing a changelog in the OP though.

**Comment by TechDragon610** [comment]
what's the procedure for making the changelog? not exactly a stellar change.

**Comment by Glorken** [comment]
When I read the PR title, I thought that an immortal Ice King had been previously added, and that this PR would allow him to die.

**Comment by Tastyfish** [comment]
:cl:[Name you want to be seen as on the changelog, will default to github name if not included]
rscadd: Ice now melts into water blah blah
/:cl:

An example in #4547

**Comment by TechDragon610** [comment]
tah da. no more ice king slaying hopes.

**Comment by Glorken** [comment]
Tech, I think tastyfish meant more like
:cl: TechDragon610
rscadd: ice melts!
/:cl:

The :cl: is made by (colon)cl(colon)

**Comment by Ar3nn** [comment]
@TechDragon610 They're talking about ice kings because you said Ice Regent in the title.

**Comment by TechDragon610** [comment]
this needs editing so the ice is set to melt at a higher temperature or spawns at a lower one than room temperature because as is, the ice reagent is pretty much not a thing because it melts on creation.

**Pull request merged**